src/components/AVPlayer/AVPlayer.js

```diff
@@ -879,4 +886,12 @@ class AVPlayer extends Component {
   }
 }
 
-export default withNamespaces()(withMediaProps(withRouter(AVPlayer)));
+export const mapState = () => ({});
```

Comment:
What do you need this for? Especially exported? Just use {} in connect()

Reply:
Fixed.